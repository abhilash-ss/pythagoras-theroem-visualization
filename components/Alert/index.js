import React from "react";

function Alert(props) {
  return (
    <div className="alert">
      <p>{props.message}</p>
      <style jsx>
        {`
          .alert {
            padding: 0px 50px;
            background: #edb2b2;
            color: red;
            margin: 10px;
            border: 1px solid red;
            border-radius: 14px;
          }
        `}
      </style>
    </div>
  );
}

export default Alert;
