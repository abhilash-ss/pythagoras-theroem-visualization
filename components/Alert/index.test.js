import React from "react";
import { shallow } from "enzyme";
import Alert from "./index";

describe("Alert", () => {
  let props;
  beforeEach(() => {
    props = {
      message: "error message"
    };
  });
  it("should render without crash", () => {
    const component = shallow(<Alert {...props} />);
    expect(component).toMatchSnapshot();
  });
});
