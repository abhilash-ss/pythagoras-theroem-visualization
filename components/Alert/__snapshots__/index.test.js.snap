// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render without crash 1`] = `ShallowWrapper {}`;
